backEnd/main.py

```diff
@@ -1,12 +1,23 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, Depends, status, HTTPException
+from fastapi.security import OAuth2AuthorizationCodeBearer, OAuth2, OAuth2PasswordRequestForm
 from pydantic import BaseModel
-
-#from routes import auth, course, user
+from routes import auth
+from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
 
-#app.include_router(auth.router, prefix="/api/v1/auth", tags=["Authentication"])
-# Uncomment if you have course and user implemented
+
+app.add_middleware(
```

Comment:
The app configuration and service registration will likely increase over time, so it would be a good idea to encapsulate these in a separate function, such as ```register_service()```, to keep the main application setup clean and organized.